omero/sysadmins/windows/server-installation.txt

```diff
@@ -124,6 +127,10 @@ The following are optional depending on what services you require:
       - Functionality
       - Downloads
 
+    * - Django (1.8.5)
```

Comment:
Django is not additional library really. I think is enough to mention pip install on IIS page.